src/gui/render/gl_enums.hpp

```
@@ -73,23 +181,225 @@ constexpr inline std::string
+            return "GL_INT_IMAGE_BUFFER"; // 	iimageBuffer
+        case GPUDataType::INT_IMAGE_1D_ARRAY:
+            return "GL_INT_IMAGE_1D_ARRAY"; // 	iimage1DArray
+        case GPUDataType::INT_IMAGE_2D_ARRAY:
+            return "GL_INT_IMAGE_2D_ARRAY"; // 	iimage2DArray
+        case GPUDataType::INT_IMAGE_2D_MULTISAMPLE:
+            return "GL_INT_IMAGE_2D_MULTISAMPLE"; // 	iimage2DMS
+        case GPUDataType::INT_IMAGE_2D_MULTISAMPLE_ARRAY:
+            return "GL_INT_IMAGE_2D_MULTISAMPLE_ARRAY"; // 	iimage2DMSArray
+        case GPUDataType::UNSIGNED_INT_IMAGE_1D:
+            return "GL_UNSIGNED_INT_IMAGE_1D"; // 	uimage1D
+        case GPUDataType::UNSIGNED_INT_IMAGE_2D:
+            return "GL_UNSIGNED_INT_IMAGE_2D"; // 	uimage2D
+        case GPUDataType::UNSIGNED_INT_IMAGE_3D:
+            return "GL_UNSIGNED_INT_IMAGE_3D"; // 	uimage3D
+        case GPUDataType::UNSIGNED_INT_IMAGE_2D_RECT:
+            return "GL_UNSIGNED_INT_IMAGE_2D_RECT"; // 	uimage2DRect
+        case GPUDataType::UNSIGNED_INT_IMAGE_CUBE:
+            return "GL_UNSIGNED_INT_IMAGE_CUBE"; // 	uimageCube
+        case GPUDataType::UNSIGNED_INT_IMAGE_BUFFER:
+            return "GL_UNSIGNED_INT_IMAGE_BUFFER"; // 	uimageBuffer
+        case GPUDataType::UNSIGNED_INT_IMAGE_1D_ARRAY:
+            return "GL_UNSIGNED_INT_IMAGE_1D_ARRAY"; // 	uimage1DArray
+        case GPUDataType::UNSIGNED_INT_IMAGE_2D_ARRAY:
+            return "GL_UNSIGNED_INT_IMAGE_2D_ARRAY"; // 	uimage2DArray
+        case GPUDataType::UNSIGNED_INT_IMAGE_2D_MULTISAMPLE:
+            return "GL_UNSIGNED_INT_IMAGE_2D_MULTISAMPLE"; // 	uimage2DMS
+        case GPUDataType::UNSIGNED_INT_IMAGE_2D_MULTISAMPLE_ARRAY:
+            return "GL_UNSIGNED_INT_IMAGE_2D_MULTISAMPLE_ARRAY"; // 	uimage2DMSArray
+        case GPUDataType::UNSIGNED_INT_ATOMIC_COUNTER:
+            return "GL_UNSIGNED_INT_ATOMIC_COUNTER"; //
         default:
-            abort();
+            return "Not Implemented";
```

No you should throw an error so you know to add it